webvtt.go

```diff
@@ -123,7 +125,9 @@ func ReadFromWebVTT(i io.Reader) (o *Subtitles, err error) {
 		// Empty line
 		case len(line) == 0:
 			// Reset block name
-			blockName = ""
+			if blockName != webvttBlockNameStyle || len(styleLines) == 0 {
```

Comment:
Why the ` || len(styleLines) == 0` ? 🤔

Reply:
Because there could be new line in CSS, like this:
```
STYLE
::cue {
  background-image: linear-gradient(to bottom, dimgray, lightgray);

  color: papayawhip;
}
```
So if we are in the middle of a CSS, we will not reset the blockName, styleLines will be empty if we meet the end of CSS before

Reply:
👍